Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'puma', '~> 5.0'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
-
+gem 'devise', '~> 4.2'
```

Comment:
As you are creating API only application, and devise by default assumes app with views etc - it would be best to add and follow guide from here: https://github.com/lynndylanhurley/devise_token_auth